src/Makefile

```diff
@@ -70,6 +70,10 @@ ifeq ($(USE_JEMALLOC),no)
 	MALLOC=libc
 endif
 
+ifeq ($(USE_MEMKIND),yes)
```

Comment:
we can let the memkind as the default malloc and update the read me.

Reply:
Done. Memkind is used as default allocator